pytorch_lightning/trainer/auto_mix_precision.py

```diff
@@ -22,14 +22,20 @@ class TrainerAMPMixin(ABC):
     def init_amp(self, use_amp):
         # TODO: remove in v 0.8.0
         if self.use_native_amp:
-            rank_zero_warn("`amp_level` has been deprecated since v0.7.4 "
-                           "(native amp does not require it)"
-                           " and this argument will be removed in v0.8.0", DeprecationWarning)
+            rank_zero_warn(
+                "`amp_level` has been deprecated since v0.7.4 "
+                "(native amp does not require it)"
+                " and this argument will be removed in v0.8.0",
```

Comment:
```suggestion
                "`amp_level` has been deprecated since v0.7.4 (native amp does not require it)"
                " and this argument will be removed in v0.8.0",
```